crates/server/src/policy/record/authorization.rs

```diff
@@ -0,0 +1,63 @@
+use super::{RecordPolicy, RecordPolicyError, RecordPolicyResult};
+use anyhow::{bail, Result};
+use std::collections::{HashMap, HashSet};
+use warg_crypto::signing::KeyID;
+use warg_protocol::{package::PackageRecord, registry::PackageId, ProtoEnvelope};
+use wasmparser::names::KebabStr;
+
+/// A policy that ensures a published record is authorized by
+/// the key signing the record.
+#[derive(Default)]
+pub struct AuthorizedKeyPolicy {
```

Comment:
Thoughts on making this `Serialize + Deserialize` so we can pull it from a config file?

Reply:
I think it'd be best addressed in a follow-up PR to properly represent a server configuration as a single file that can be deserialized to enable the policies (with their specific configurations).

I purposely punted on that (i.e. there's no way to enable policies in the server CLI) as I would want to have a proper discussion around configuration file format.